### DISCUSSION TITLE
PAGView在ViewPager2中的正确使用方式是什么

### DISCUSSION THREAD

**Discussion by PJYGit** [Tencent/libpag#1664] [Q&A] [answered]
SDK version 4.3.3

ViewPager2中：
pos = 0, pag动画播放进度0.5后滑动到下一个 pos = 1；
然后滑动回上一个pos = 0，再次调用PAGView.play()，发现还是从progress=0.5开始播放

如果我想每次都重新播放怎么解决？
setProgress(0)开销高吗？

**Comment by kevingpqi123** [comment, marked as answer]
应该是你动画进度设置的有问题，切换的时候执行 stop（），setProgress(), 然后 play
setProgress 仅仅是设置进度，没啥开销，开销高的时候设置进度后的渲染首帧（需要重新构建缓存）
